woocommerce/woocommerce
# Fix breadcrumb back icon color to follow the admin theme color

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:
This PR fixes the breadcrumb back icon color issue described in https://github.com/woocommerce/woocommerce-gateway-stripe/issues/2680

- Appended the variation selector `&#xfe0e;` with the back icon unicode character. It prevents the character from being rendered as an image and hence allows the icon to pick up the `var(--wp-admin-theme-color)`.

### How to test the changes in this Pull Request:
- Go to `<your_site/wp-admin/profile.php` and choose an Admin Color Scheme except for `default`. (I have used `Modern` in the screenshots)
- Go to **WooCommerce > Settings > Payments**.
- Choose any of the available options (Cash on delivery/Direct bank transfer/Stripe/WooPayment etc.) and click to enter the settings.
- Check the back icon and it should follow the admin theme color.

**Before:**

![cod before](https://github.com/woocommerce/woocommerce/assets/33387139/7b2a6a73-b5dd-4ff7-b127-6ac3a8132e25)


**After:**

![after cod](https://github.com/woocommerce/woocommerce/assets/33387139/22d1782b-7cfa-4b1d-b3d2-0122a5090b9c)
